src/app/CreateRecording/CustomRecordingForm.tsx

```diff
@@ -224,8 +233,10 @@ export const CustomRecordingForm = (props) => {
       <FormGroup
         label="Duration"
         isRequired
-        fieldId="recording-duration"
+        validated={durationValid}
         helperText="Time before the recording is automatically stopped"
+        helperTextInvalid="A recording may only have a non-zero integer duration"
```

Comment:
`non-zero integer` would include negatives :-)

Just say `positive` here, I think, and maybe add a bit of description about Continuous recordings never being (automatically) stopped.

Is it possible to have the number input prevent 0 from being entered? It already won't let me put in a negative number or non-numeric value.

Reply:
> Is it possible to have the number input prevent 0 from being entered? It already won't let me put in a negative number or non-numeric value.

I think this is a feature of patternfly's `TextInput` `type` prop which is set to `"number"`, which only allows positive numbers and decimals internally but maybe I can figure something out to override it.

Reply:
So, there is not really an easy way to do this that I can see. The patternfly `TextInput` prop `type='number'` forces the input to always contain a number so if a user deletes everything in the input, the input while never be empty. I'm tried
```
  const handleDurationChange = React.useCallback((evt) => {
    if (Number(evt) == 0) {
      return;
    }
    setDuration(Number(evt));
    setDurationValid(DurationPattern.test(evt) ? ValidatedOptions.success : ValidatedOptions.error);
  }, [setDurationValid, setDuration]);
```
so that the input just doesn't allow the full number 0 to be in the input, but it feels annoying if a user tries to clear the field to type a new number since it wouldn't allow it to be cleared at all.

So one way to get past this is to just not use the `type` prop and do manual `onChange` checking so that users can actually clear the field, but still manually only allow numbers to be input. 

Thoughts or clarifications?

Reply:
Okay, don't worry about it then. Removing the `type` prop would make things even worse I think, since we would need to handle non-numeric inputs and things too.